src/lib/metrics.ts

```diff
@@ -85,10 +85,10 @@ export default class MetricsMonitor {
             maxAgeSeconds: 600,
             ageBuckets: 5,
         });
-        const operationDuration = createSummary({
-            name: 'operation_duration_seconds',
-            help: 'Operation duration time',
-            labelNames: ['operationId'],
+        const methodDuration = createSummary({
```

Comment:
why do you rename it to method? Methods exist on objects. If we wanted to time a regular function method wouldn't make sense anymore.